buildtools/fsyacc/fsyacc.fsproj

```diff
@@ -8,6 +8,11 @@
     <RollForward Condition="'$(BUILDING_USING_DOTNET)' == 'true'">LatestMajor</RollForward>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <PublishReadyToRun>true</PublishReadyToRun>
```

Comment:
My preference is for the project to specify this stuff rather than directory.props and targets, it makes it clearer at least to me.  But I can move it if you really think it's better that way.